README.md

```diff
@@ -42,14 +42,29 @@ Then run:
 ```
 docker build -t rodekruis510/typhoonibf .
 ```
+Create a docker container
```

Comment:
The README needs to be updated to say that all docker commands should be run from the IBF-Typhoon-model directory